web/packages/teleport/src/Roles/RoleEditor/StandardEditor/Options.tsx

```diff
@@ -59,156 +60,163 @@ export const Options = memo(function Options({
   const sshPortForwardingModeId = `${id}-ssh-port-forwarding-mode`;
 
   return (
-    <OptionsGridContainer
-      border={1}
-      borderColor={theme.colors.interactive.tonal.neutral[0]}
-      borderRadius={3}
-      p={3}
-    >
-      <OptionsHeader>Global Settings</OptionsHeader>
+    <Flex flexDirection="column" gap={3}>
+      Additional advanced settings
```

Comment:
Adding horizontal margin to match the padding applied to the `OptionsGridContainer` would make these descriptions fit much better IMO. Should also be wrapped in `<Text>` to make sure we don't get any weird default-OS-font behaviour.

Reply:
hrmm, i was under the impression we had a global css that defined the fonts?

Reply:
I don't think so? I've run into several cases before where I've added text directly inside a <Button> or <Flex> and it looks odd on some browsers / OS'... possible that's since changed, though.

Don't want to block on it, but I do think adding some horizontal spacing to these descriptions at least would be good.

Reply:
I have never seen anything being overridden this way. Perhaps buttons and other native controls, yes, because they are always strongly styled by the user agent stylesheet and frequently need to be overridden, but never inside regular text. I applied your margin suggestion, though; it looks much better this way.